src/Type/DatetimeType.php

```diff
@@ -38,6 +38,6 @@ private function isValidDatetime(string $value): bool
         $date = DateTime::createFromFormat($this->datetimeFormat, $value, new DateTimeZone($this->datetimeZone));
         $errors = DateTime::getLastErrors();
 
-        return $date && $errors['warning_count'] === 0 && $errors['error_count'] === 0;
+        return $date && !$errors;
```

Comment:
This breaks the library: https://3v4l.org/hFvIJ
I do not know why php.net tells us that the function would return `false`, but it does not. Neither in 7.4, nor 8.0, nor 8.1 (see the 3v4l)